src/app/lhc-item-gird/lhc-item-gird.component.html

```diff
@@ -0,0 +1,51 @@
+<div *ngIf="item" class="lhc-layout-grid lf-table-item {{lhcDataService.getSiblingStatus(item)}}">
+  <div class="lf-form-grid-table-title" *ngIf="!formLevel">
+    <lhc-item-question-text [item]="item"></lhc-item-question-text>
+  </div>
+
+  <table class="lf-form-grid-table lf-form-table">
+    <colgroup>
+      <col class="lf-question">
+      <col *ngFor="let answer of item.items[0].answers">
```

Comment:
It looks like you have implemented the "matrix" ("table" in FHIR itemControl), which we already have. Could you include some test data files so we can take a look at the rendered questionnaires?